src/activities/hoc2019/P5Canvas.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {PropTypes} from 'react';
+const P5 = require('../../utils/loadP5');
+//import {sketch} from '../../utils/sketches';
+import Col from 'react-bootstrap/lib/Col';
+import Button from 'react-bootstrap/lib/Button';
+
+export default class P5Canvas extends React.Component {
+  static propTypes = {
+    fish_data: PropTypes.object.isRequired,
+    canvasId: PropTypes.string.isRequired,
+    sketch: PropTypes.func.isRequired,
+    addExample: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {p5: null};
+  }
+
+  componentDidMount() {
+    const p5 = new P5(this.props.sketch, this.props.canvasId);
+    this.setState({p5: p5});
```

Comment:
since the key/value have the same name, we can shorten this line to:
```suggestion
    this.setState({p5});
```

Reply:
Ah I keep forgetting about this!